observer/observer.go

```diff
@@ -185,3 +214,13 @@ func (o *Observer) RegisterCallback(name string, callback func()) {
 		o.observedJobs[name] = event
 	}
 }
+
+func incFailure(namespace string) {
+	metricsFailureCounter.WithLabelValues(namespace).Inc()
+	metricsTotalCounter.WithLabelValues(namespace).Inc()
+}
+
+func incSuccess(namespace string) {
+	metricsSuccessCounter.WithLabelValues(namespace).Inc()
+	metricsTotalCounter.WithLabelValues(namespace).Inc()
+}
```

Comment:
Let's make the method names a little more descriptive
```suggestion
func incFailureCounters(namespace string) {
	metricsFailureCounter.WithLabelValues(namespace).Inc()
	metricsTotalCounter.WithLabelValues(namespace).Inc()
}

func incSuccessCounters(namespace string) {
	metricsSuccessCounter.WithLabelValues(namespace).Inc()
	metricsTotalCounter.WithLabelValues(namespace).Inc()
}
```